### DISCUSSION TITLE
[Support]: Question on History Time Bar Slider Colors

### DISCUSSION THREAD

**Discussion by moogoos** [blakeblackshear/frigate#15108] [General Support] [answered]
### Describe the problem you are having

What is the significance of the maroon color in the history timeline slider? Also I assume the yellow horizontal lines are motion detections, size based on movement intensity? See image.

### Version

15-beta1

### What browser(s) are you using?

Chrome

### Frigate config file

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Frigate stats

```json
NA
```

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
NA
```

### Object Detector

OpenVino

### Network connection

Wired

### Camera make and model

Hikvision

### Any other information that may be helpful

![Screenshot 2024-11-20 at 3 12 22 PM](https://github.com/user-attachments/assets/004fa73b-93e3-4080-98cc-f4bc9e8dc639)

**Comment by NickM-27** [comment, marked as answer]
The background represents that type of activity. So red background means alert and orange background means detection. Yes, yellow bars are motion
